src/Microsoft.AspNet.Server.Kestrel/Http/FrameRequestHeaders.cs

```diff
@@ -21,11 +21,11 @@ public Enumerator GetEnumerator()
 
         public partial struct Enumerator : IEnumerator<KeyValuePair<string, StringValues>>
         {
-            private FrameRequestHeaders _collection;
-            private long _bits;
+            private readonly FrameRequestHeaders _collection;
+            private readonly long _bits;
             private int _state;
             private KeyValuePair<string, StringValues> _current;
-            private bool _hasUnknown;
+            private readonly bool _hasUnknown;
             private Dictionary<string, StringValues>.Enumerator _unknownEnumerator;
```

Comment:
For whatever reason some tests (`Microsoft.AspNet.Server.KestrelTests.FrameRequestHeadersTests.EntriesCanBeEnumerated` and `Microsoft.AspNet.Server.KestrelTests.FrameRequestHeadersTests.KeysAndValuesCanBeEnumerated`) fail when I set `_unknownEnumerator` to `readonly`.